src/core/lv_obj_pos.c

```diff
@@ -795,18 +795,18 @@ void lv_obj_move_children_by(lv_obj_t * obj, lv_coord_t x_diff, lv_coord_t y_dif
     }
 }
 
-void lv_obj_transform_point(const lv_obj_t * obj, lv_point_t * p, bool recursive, bool inv)
+void lv_obj_transform_point(const lv_obj_t * obj, lv_point_t * p, lv_point_t * p1, bool recursive, bool inv)
```

Comment:
```suggestion
void lv_obj_transform_point(const lv_obj_t * obj, lv_point_t * p, lv_point_t * pref, bool recursive, bool inv)
```

Reply:
I can’t think of a better name, so let's use this.

Reply:
"this" means "pref" or "p1"? :slightly_smiling_face:

Reply:
I mean "reference point" (should have been "p_ref") but "prev_p" is fine too.